tests/Contracts/CancelTasksQueryTest.php

```diff
@@ -13,22 +13,22 @@ public function testSetTypes(): void
     {
         $data = (new CancelTasksQuery())->setTypes(['abc', 'xyz']);
 
-        $this->assertEquals(['types' => 'abc,xyz'], $data->toArray());
+        static::assertEquals(['types' => 'abc,xyz'], $data->toArray());
```

Comment:
Can you also use 'asserSame` everywhere where possible?

Reply:
Sure, right after #613. It prevents me from making this change.

Reply:
Done.